package.json

```diff
@@ -29,6 +31,7 @@
     "react-map-gl": "^7.0.19",
     "react-redux": "^8.0.4",
     "react-router-dom": "^6.6.1",
+    "react-table": "^7.8.0",
```

Comment:
Why do we need both this and `@tanstack/react-table`? Also, why does this PR add in `mapbox-gl` as a dependency?

Reply:
We can remove "react-table": "^7.8.0" and keep the tanstack version. The mapbox-gl was a dependency from tanstack I believe.

Reply:
Also added screenshots